crates/stackable-versioned-macros/fixtures/inputs/default/module.rs

```diff
@@ -0,0 +1,22 @@
+#[versioned(
+    version(name = "v1alpha1"),
+    version(name = "v1"),
+    version(name = "v2alpha1")
+)]
+// ---
+pub(crate) mod versioned {
+    pub struct Foo {
+        bar: usize,
+
+        #[versioned(added(since = "v1"))]
+        baz: bool,
+
+        #[versioned(deprecated(since = "v2alpha1"))]
+        deprecated_foo: String,
+    }
+
+    #[versioned]
+    pub struct Bar {
+        baz: String,
+    }
```

Comment:
What does this `#[versioned]` do inside the versioned module?

Reply:
> What does this `#[versioned]` do inside the versioned module?

This basically tests that nothing funky is being generated when an empty `#[versioned]` attribute is attached.

> Or rather, why does `struct Foo` not have one, but the items inside have actions?

Every supported item inside a versioned module is _automatically_ versioned and thus supports actions to be attached to fields. Attaching the attribute to contained containers enables further customization, but is not required.

Reply:
> > What does this `#[versioned]` do inside the versioned module?
> 
> This basically tests that nothing funky is being generated when an empty `#[versioned]` attribute is attached.

Maybe we can add a dev comment for future reference?

```suggestion
    // The following attribute is just to ensure no strange behavior occurs.
    #[versioned]
    pub struct Bar {
        baz: String,
    }
```

Reply:
> So it's a NoOp?

It is.

> Maybe we can add a dev comment for future reference?

Sure, that makes sense.

Reply:
Done in 626915a